.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,12 @@
-**Issue #:** 
+**Issue #:**
 
 **Description of changes:**
 
 **Testing**
 
 1. Have you successfully run `npm run build:release` locally?
 2. How did you test these changes?
+3. Does this change add/update any APIs or change wire protocol? If yes, has that been reviewed & approved?
```

Comment:
- nit: How about rephrasing using `you` as in the first two questions?
  e.g. `Do you add or update API, including the wire protocol?`
- nit: `reviewed & approved` => `reviewed and approved`

Reply:
I just understand that we are asking two questions.
1. Add or update APIs
    I think we can be more specific here, e.g. `Do you add, modify, or delete public API definitions?` (or more specific by saying public method signatures and return type)
2. Make a wire protocol-related change 
   You can add one more sentence for this: `Do you change the wire protocol, e.g. the request method?`

Do you add, modify, or delete public API definitions? Do you change the wire protocol, e.g. the request method? If yes, ...